### ISSUE TITLE
[Research] OUI Compliance audit of the `vis_type_markdown` plugin

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4097]
Tracking issue for OUI compliance audit of the `vis_type_markdown` plugin.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

<details>
  <summary>Related files</summary>

  size (bytes) | file | notes
---|---|---
365 | [src/plugins/vis_type_markdown/public/markdown_vis.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_markdown/public/markdown_vis.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.

**Comment by curq** [comment]
@BSFishy Can I please take this one?

**Comment by joshuarrrr** [comment]
Sure @curq I assigned you.

**Comment by curq** [comment]
@BSFishy finished https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4347
